src/dguweb/web/static/elm/SearchBox.elm

```diff
@@ -182,24 +182,17 @@ viewForm =
                 )
             )
     in
-        Html.form
-            [ action "/search"
-            , method "GET"
-            ]
-            [ Html.fieldset []
-                [ Html.input
-                    [ id "q"
-                    , name "q"
-                    , type' "text"
-                    , placeholder "Search for data"
-                    , class "form-control search"
-                    , autocomplete False
-                    , onInput Lookup
-                    , onWithOptions "keydown" keydownOptions keydownDecoder
-                    ]
-                    []
-                ]
+        Html.input
```

Comment:
Is there anyway that this might be able to obtain the value for ```query``` here. If I search on the homepage, and press enter without selecting from the dropdown, any term I entered on the homepage is not shown in the input.